src/about.js

```diff
@@ -11,7 +11,8 @@ module.exports = projectLocation => {
       version: packageJson.version,
       description: packageJson.description || '<no description>',
       author: packageJson.author || '<no author>',
-      license: packageJson.license || 'AGPL-v3.0'
+      license: packageJson.license || 'AGPL-v3.0',
+      flowEditorDisabled: Boolean(process.env.BOTPRESS_FLOW_EDITOR_DISABLED)
```

Comment:
![image](https://user-images.githubusercontent.com/1315508/38163224-15e5d494-350e-11e8-9380-c2ee4af4bfab.png)

See the `yn` module to parse env strings